config/application.rb

```diff
@@ -4,25 +4,29 @@
 require "active_model/railtie"
 require "active_job/railtie"
 require "active_record/railtie"
+require "active_storage/engine"
```

Comment:
Do we actually need to load active_storage if we're not using it?

Reply:
Let's load it, so we are one step closer to use it.